composer.json

```diff
@@ -1,56 +1,56 @@
 {
-	"name": "hitmeister/api-sdk",
-	"description": "Real.de onlineshop API SDK for PHP",
-	"type": "library",
-	"version": "1.21.0",
-	"authors": [
-		{
-			"name": "Real.de Onlineshop Developers",
-			"email": "devops@real-digital.de"
-		}
-	],
-	"license": "MIT",
-	"minimum-stability": "stable",
-	"keywords": [
-		"hitmeister",
-		"sdk",
-		"api",
-		"real.de",
-		"real.digital",
-		"real-digital"
-	],
-	"homepage": "https://www.real.de/api/v1/",
-	"support": {
-		"email": "devops@real-digital.de",
-		"source": "https://github.com/hitmeister/api-sdk-php",
-		"issues": "https://github.com/hitmeister/api-sdk-php/issues"
-	},
-	"require": {
-		"php": ">=5.4.0",
-		"ext-curl": "*",
-		"guzzlehttp/ringphp": "^1.1",
-		"psr/log": "^1.0"
-	},
-	"require-dev": {
-		"mockery/mockery": "^0.9.4",
-		"phpunit/phpunit": "^4.8",
-		"satooshi/php-coveralls": "^1.0",
-		"memio/memio": "^1.1"
-	},
-	"suggest": {
-		"monolog/monolog": "Allows more advanced logging of the application flow"
-	},
-	"autoload": {
-		"psr-4": {
-			"Hitmeister\\Component\\Api\\": "src/"
-		}
-	},
-	"autoload-dev": {
-		"psr-4": {
-			"Hitmeister\\Component\\Api\\Tests\\": "tests/"
-		}
-	},
-	"scripts": {
-		"test": "phpunit"
-	}
+    "name": "hitmeister/api-sdk",
+    "description": "Real.de onlineshop API SDK for PHP",
+    "type": "library",
+    "version": "1.21.0",
+    "authors": [
+        {
+            "name": "Real.de Onlineshop Developers",
+            "email": "opensource@real-digital.de"
+        }
+    ],
+    "license": "MIT",
+    "minimum-stability": "stable",
+    "keywords": [
+        "hitmeister",
+        "sdk",
+        "api",
+        "real.de",
+        "real.digital",
+        "real-digital"
+    ],
+    "homepage": "https://www.real.de/api/v1/",
+    "support": {
+        "email": "opensource@real-digital.de",
+        "source": "https://github.com/hitmeister/api-sdk-php",
+        "issues": "https://github.com/hitmeister/api-sdk-php/issues"
+    },
+    "require": {
+        "php": ">=5.4.0",
+        "ext-curl": "*",
+        "guzzlehttp/ringphp": "^1.1",
+        "psr/log": "^1.0"
+    },
+    "require-dev": {
+        "mockery/mockery": "^0.9.4",
+        "phpunit/phpunit": "^4.8",
+        "php-coveralls/php-coveralls": "^1.1",
+        "memio/memio": "^1.1"
+    },
+    "suggest": {
+        "monolog/monolog": "Allows more advanced logging of the application flow"
+    },
+    "autoload": {
+        "psr-4": {
+            "Hitmeister\\Component\\Api\\": "src/"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "Hitmeister\\Component\\Api\\Tests\\": "tests/"
+        }
+    },
+    "scripts": {
+        "test": "vendor/bin/phpunit"
```

Comment:
Composer automatically looks inside `vendor/bin`

Reply:
According to the [documentation](https://getcomposer.org/doc/articles/scripts.md) composer is quite smart:

>no matter if the phpunit binary is actually in vendor/bin/phpunit or bin/phpunit it will be found and executed.

Yet I prefer to point its path explicitly to exclude complains from IDE side about missing the file.